frontend/ui/uimanager.lua

```diff
@@ -86,6 +86,25 @@ function UIManager:init()
               self.event_handlers["Suspend"]()
             end
         end
+        if not G_reader_settings:readSetting("ignore_power_sleepcover") then
+            self.event_handlers["SleepCoverClosed"] = function()
+                Device.is_cover_closed = true
+                self.event_handlers["Suspend"]()
+            end
+            self.event_handlers["SleepCoverOpened"] = function()
+                Device.is_cover_closed = false
+                self.event_handlers["Resume"]()
+            end
+        else
+            -- Closing/opening the cover will still wake up the device, so we
+            -- need to put it back to sleep if we are in screen saver mode
+            self.event_handlers["SleepCoverClosed"] = function()
```

Comment:
Yes, I do not have a sleep cover, I cannot quite tell. Though I can use my kindle to turn kobo on / off without ignore_power_sleepcover.